onnxruntime/core/platform/windows/env.cc

```diff
@@ -397,6 +380,10 @@ class WindowsEnv : public Env {
   common::Status GetCanonicalPath(
       const PathString& path,
       PathString& canonical_path) const override {
+#if WINAPI_FAMILY_PARTITION(WINAPI_PARTITION_APP)
+    canonical_path = path;
+    return Status::OK();
+#else
```

Comment:
I'll revisit this later to see if there's any APIs to do this in an appcontainer